src/compiler/types/tests/stencil-types.spec.ts

```diff
@@ -71,6 +71,24 @@ describe('stencil-types', () => {
         expect(actualTypeName).toBe(expectedTypeName);
       });
 
+      it('returns the provided type for imports without the resolved file', () => {
```

Comment:
this test explicitly tests [this code](https://github.com/ionic-team/stencil/pull/5026/files#diff-9520ec3dc9b12aac1a3735d5bc8a2808c1cf32a7c560ab81874b9e6aa10b9f37R60-R62), which was previously existing (I know, right?). 

It's [this existing test](https://github.com/ionic-team/stencil/pull/5026/files#diff-f3b145b0c322c69d725a877f98b882fbe836e81787ee2043fa42d182841f82b2L57-L72) that covers the new conditional, [found here](https://github.com/ionic-team/stencil/pull/5026/files#diff-9520ec3dc9b12aac1a3735d5bc8a2808c1cf32a7c560ab81874b9e6aa10b9f37R56-R59)